test/cli/test_cli_dicom_to_nifti.py

```diff
@@ -19,8 +19,7 @@ def test_cli_dicom_to_nifti(test_dcm2niix_installation):
         path_dicoms = os.path.join(__dir_testing__, 'dicom_unsorted')
         path_nifti = os.path.join(tmp, 'nifti')
         subject_id = 'sub-test'
-        result = runner.invoke(dicom_to_nifti_cli, ['--input', path_dicoms, '--output', path_nifti,
-                                                    '--subject', subject_id])
+        result = runner.invoke(dicom_to_nifti_cli, ['-i', path_dicoms, '-o', path_nifti, '--subject', subject_id])
```

Comment:
We should add the `catch_exception=False` option

Reply:
From what I understand from the [Click documentation](https://click.palletsprojects.com/en/8.0.x/api/), not catching the exceptions would make our test less robust so I would leave it this way.